config/management/operational/src/test_helper.rs

```diff
@@ -116,6 +119,44 @@ impl OperationalTool {
         let command = Command::from_iter(args.split_whitespace());
         command.validate_transaction()
     }
+
+    pub fn validator_config(
+        &self,
+        account_address: AccountAddress,
+    ) -> Result<ValidatorConfig, Error> {
+        let args = format!(
+            "
+                {command}
+                --host {host}
+                --account-address {account_address}
+        ",
+            command = command(TOOL_NAME, CommandName::ValidatorConfig),
+            host = self.host,
+            account_address = account_address,
+        );
+
+        let command = Command::from_iter(args.split_whitespace());
+        command.validator_config()
+    }
+
+    pub fn validator_set(
+        &self,
+        account_address: AccountAddress,
+    ) -> Result<Vec<ValidatorInfo>, Error> {
+        let args = format!(
+            "
+                {command}
+                --host {host}
+                --account-address {account_address}
```

Comment:
@zekun000, let's use consistent naming across the tools, I think you call this account_to_add?